src/test/java/com/epam/training/auction_backend/services/AuctionsServiceImplTestIT.java

```diff
@@ -0,0 +1,105 @@
+package com.epam.training.auction_backend.services;
+
+import com.epam.training.auction.common.AuctionTransferObject;
+import com.epam.training.auction.common.UsersService;
+import com.epam.training.auction_backend.entity.Auction;
+import com.epam.training.auction_backend.entity.User;
+import com.epam.training.auction_backend.util.CommonIntegrationTestSuit;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import javax.transaction.Transactional;
+import java.util.List;
+
+public class AuctionsServiceImplTestIT extends CommonIntegrationTestSuit {
+
+    @Autowired
+    private AuctionRepository auctionRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UsersService usersService;
+
+    private AuctionsServiceImpl auctionsService;
+
+    @Before
+    public void setUp() {
+        auctionsService = new AuctionsServiceImpl(auctionRepository);
+    }
+
+    @Test
+    @Transactional
+    public void getArchivedAuctions() {
+        final int expectedArchivedAuctionsSize = 2;
+
+        // create seller and winner for auctions
+        User seller = createUser("Seller", "Seller");
```

Comment:
Yes we can, but we should have in @Before method only that thing that we will need for evry test method now and in future. I'm not sure that we will need seller in every method. The thing which we will need in evry method for 100% is already in @Before method.